src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVUtils.java

```diff
@@ -165,6 +39,13 @@ public static int iteratorSize( final LongIterator itr ) {
         return Collections.singletonList(t).iterator();
     }
 
+    public static Collection<SVKmer> uniquify(final Collection<SVKmer> coll1, final Collection<SVKmer> coll2) {
+        final HopscotchSet<SVKmer> kmers = new HopscotchSet<>(coll1.size() + coll2.size());
```

Comment:
Without looking at the uses... I guess you again simply added coll2 elements into coll1 and return coll1.

Reply:
Since is public you must check arguments are legal (non null).

Reply:
done checking. If second is added to 1st, the 1st is modified, unlike what it is doing now (copying from the 1st).

Reply:
My understanding is that you don't want to make the change because you think it will break it? I don't think it would but I guess I have to find the reference supporting my claim.

Reply:
Not necessarily breaking it, but that would change the behavior (and of course the signature) of the function.